packages/accounts/plugindefs/types.ts

```diff
@@ -0,0 +1,10 @@
+import type { Abi, Address, Chain } from "viem";
+
+export type PluginGenConfig = {
+  abi: Abi;
+  address: Address;
+  chain: Chain;
+  name: string;
+  // TODO: need to make this configurable to run in CI without requiring this
+  rpcUrl?: string;
```

Comment:
what do you mean configurable here?

Reply:
I think I forgot to remove the TODO because it is optional now

Reply:
the idea is that we shouldn't have to hardcode an RPC URL here. This is useful for testing locally if you want to hit a local anvil instance

Reply:
is there a better way to write this type so it's dependent on environment to see if rpcUrl is defined?